src/components/LogoFooter.tsx

```diff
@@ -8,13 +11,54 @@ import classnames, {
   flexDirection,
   justifyContent,
   margin,
+  position,
   space,
   zIndex,
 } from 'classnames/tailwind'
 
+const Backgrounds = ({ inView }: { inView: boolean }) => (
+  <BackgroundsAnimated inView={inView}>
```

Comment:
Should we move all `Background`'s components to a separate dir? Like `components/Background/LogoFooter`, for instance

Reply:
Done :)